### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by mesnico** [yiling2018/saem#4]
Thank you for the nice repo. I am trying to evaluate your model on a different metric. 
Can you please share the pre-trained models? It would be really appreciated. Thank you!
